Pipfile

```diff
@@ -31,7 +31,7 @@ types-urllib3 = "*"
 pre-commit = "*"
 flake8 = "*"
 dlint = "*"
-mypy = "*"
+mypy = "==1.13.0"
```

Comment:
why did you lock the version? it will prevent getting those updates which are important

Reply:
I don't want in this PR to fix mypy issues from other files due to new version

Reply:
I also don't want you to change it so we will be locked, as no one will update mypy if you do so. 
If you wish to separate the fixes from this PR, you can open another one which only updates mypy to the latest version and fixes the relevant issues